### ISSUE TITLE
cursor position displaced

### ISSUE THREAD

**Issue by schelbertgeorg** [asciidocfx/AsciidocFX#411]
cursor position is displaced by one step to the right (in the editor window, not talking about the preview).

**Comment by rahmanusta** [comment]
Do you use such special unicode characters? How can I reproduce it ?

**Comment by schelbertgeorg** [comment]
yes, indeed, I am using special characters as it often happens in the diversity of languages outside the Englisch bubble. Also looks like the above mentioned problem has to do with encoding and special characters as the latter were rendered badly in the pdf. Have no idea how to reproduce it. PS: using asciidoxfx on windows 10.

**Comment by schelbertgeorg** [comment]
Hi, I think I have understood the problem now. It occurs if you copy text into asciidocfx from another text editor like Word. gosh - it would be helpful to get malformed special characters highlithed.

**Comment by wouter-palmsens** [comment]
I also have this problem and it seems related to certain (but not all) Unicode characters. For example, the check mark (U+2713) character has this problem.

It can be simply reproduced by typing the following comment in AsciidoxFX:

```none
// After adding the ✓ symbol, the cursor is in the wrong place :-(
```

To really understand the problem, just try to edit the last part of this sentence (e.g. try to change the word "is" to "was") and you'll probably notice that you didn't end up with what you expected.
